### DISCUSSION TITLE
Need a hint, how to ignore some payloads

### DISCUSSION THREAD

**Discussion by Burki24** [Koenkk/zigbee2mqtt#12122] [Devices]
@Koenkk 
Hi,
I need some help. I use the aqara SP-EUC01 device. Within the payload, there are some definitions, not used and needed by this device. Getting some Errors in Smcon-Sstem. So I tried to eleminate them by setting them into the ignore-part of the fromzigbee-file.

```json
{
  "consumer_connected": true,
  "consumption": 0,                      // not needed as it is depricated and not used by device
  "current": 0,
  "device_temperature": 30,
  "energy": 0,
  "illuminance": 0,                      // not needed, as it is not used by device
  "illuminance_lux": 0,                  // not needed, as it is not used by device
  "linkquality": 160,
  "power": 0,
  "power_outage_count": 1,
  "state": "OFF",
  "temperature": 30,                     // not needed, as it is included correctly as device_temperature
  "voltage": 251
}
```
I tried to add following code to "fromzigbee.js":

```js
    ignore_temperature: {
        cluster: 'aqaraOpple',
        type: ['attributeReport', 'readResponse'],
        convert: (model, msg, publish, options, meta) => null,
    }
```
And added to xiaomi.js 'model: 'SP-EUC01''
```js
fz.ignore_temperature
```
But nothing happend. Can you please help, to find the right way to fix that?

Many thanks in advance

Burkhard

**Comment by lux73** [comment]
what about `filtered_attributes:` in `configuration.yaml` or rather `devices.yaml`?

**Reply by Burki24** [reply]
thanks for the tip.

But that's a local solution. So everyone with that device need to modify the configuration.yaml / devices.yaml. But this device use this DP not in general. So it would be more logical to eleminate them inside the converter, I think.

**Comment by gonzzzales** [comment]
@Burki24 Did you found the solution? I have same problem. I add some attributes in Filtered publish attributes, but still become the same payloads
{"co2":789,"humidity":32.66,"illuminance":25299,"occupancy":false,"state_l1":"OFF","state_l2":"OFF","state_l3":"OFF","temperature":25.88,"update":{"state":"idle"},"voc":242}
{"co2":789,"humidity":32.66,"illuminance":25299,"occupancy":false,"state_l1":"OFF","state_l2":"OFF","state_l3":"OFF","temperature":25.88,"update":{"state":"idle"},"voc":242}
